frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -37,9 +47,26 @@ import {
 } from './util';
 
 function Application({ getWidgetQueryBuilder }: DashboardProps): JSX.Element {
+	const globalTime = useSelector<AppState, GlobalReducer>(
+		(state) => state.globalTime,
+	);
+
+	const { maxTime, minTime } = GetMinMax(globalTime.selectedTime, [
```

Comment:
we already have minMax time from globalTime reducer no need for this one

Reply:
Okay